packages/account/src/Sections/Profile/PhoneVerification/__test__/cancel-phone-verification-modal.spec.tsx

```diff
@@ -25,15 +26,23 @@ describe('CancelPhoneVerificationModal', () => {
         document.body.removeChild(modal_root_el);
     });
 
+    const mock_store = mockStore({});
+
     const buttons = [/Go back/, /Yes, cancel/];
 
-    it('it should render CancelPhoneVerificationModal', () => {
+    const renderComponent = () => {
         render(
-            <CancelPhoneVerificationModal
-                should_show_cancel_verification_modal
-                setShouldShowCancelVerificationModal={mockSetShowCancelModal}
-            />
+            <StoreProvider store={mock_store}>
+                <CancelPhoneVerificationModal
+                    should_show_cancel_verification_modal
+                    setShouldShowCancelVerificationModal={mockSetShowCancelModal}
+                />
+            </StoreProvider>
         );
+    };
+
+    it('it should render CancelPhoneVerificationModal', () => {
+        renderComponent();
         expect(screen.getByText(/Cancel phone number verification?/)).toBeInTheDocument();
         expect(screen.getByText(/All details entered will be lost./)).toBeInTheDocument();
         buttons.forEach(value => {
```

Comment:
just a nitpick, can we define btn before and validate next line, just for better readability. You can ignore this 🙏

Reply:
sure 😆